plugins/woocommerce/changelog/update-contributor-list

```diff
@@ -0,0 +1,5 @@
+Significance: patch
+Type: dev
+Comment: This PR simply adds contributors of WooCommerce Blocks to the contributors list.
```

Comment:
@nielslange there should be only one changelog entry right?

Reply:
There should indeed only be one changelog entry. I must have accidentally selected the option Automatically create a changelog entry from the details below. even though I've already manually added a changelog entry. I just removed the other one.